solidity/nomad-xapps/contracts/bridge/BridgeRouter.sol

```diff
@@ -377,6 +383,23 @@ contract BridgeRouter is Version0, Router {
             PRE_FILL_FEE_DENOMINATOR;
     }
 
+    /**
+     * @notice Dust the recipient.
+     * @dev Does not dust if insufficient funds, or if user has funds already
+     */
+    function _dust(address _recipient) internal {
+        if (
+            _recipient.balance <= DUST_AMOUNT &&
+            address(this).balance >= DUST_AMOUNT
+        ) {
+            // `send` gives execution 2300 gas and returns a `success` boolean.
+            // however, we do not care if the call fails. A failed call
+            // indicates a smart contract attempting to execute logic, which we
+            // specifically do not want.
+            payable(_recipient).send(DUST_AMOUNT);
```

Comment:
> which we specifically do not want

thinking about this. chewing on this.

- we deffo don't want to pass a bunch of gas, but we _could_ pass a bit more than 2300 to allow simple smart wallets, for example. 
- probably an edge case for new chains though.
- hmmmmmmmmmmmmmmmmmm

Reply:
A smart contract wallet doesn't need the dust, as it's gas is paid by some other eoa